### PR TITLE
Added expanding sections

### PR DESCRIPTION
**Pull request by mattsskea**

### CODE REVIEW

File: components/expanding-components/section.scss [hunk 1]
```diff
@@ -0,0 +1,52 @@
+.msds-expanding-section {
+  display: flex;
+  justify-content: center;
+  width: 100%;
+  // padding-top: 96px;
```
Comment (ms-Danny) [inline components/expanding-components/section.scss:5]: please remove commented code

File: components/expanding-components/section.scss [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  // padding-bottom: 80px;
+
+  &--closed {
+    display: flex;
```
Comment (ms-Danny) [inline components/expanding-components/section.scss:9]: i dont think you need display flex here as its already set on the outer selector. You only need to specify if you want to overwrite something

File: components/expanding-components/section.scss [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+      ~ .msds-expanding-section__buttons-container {
+        background: red;
+        height: 100px;
+        width: 100%;
+
+        .msds-expanding-section--closed {
+          display: none;
+        }
+
+        .msds-expanding-section--open {
```
Comment (ms-Danny) [inline components/expanding-components/section.scss:42]: i does not make sense to me why you have to specify display flex here aswell as it already has been declared on the block on line 2

File: components/expanding-components/section.scss (hunk 1 above)
Comment (mattsskea) [reply components/expanding-components/section.scss:5]: Removed

File: components/expanding-components/section.scss (hunk 2 above)
Comment (mattsskea) [reply components/expanding-components/section.scss:9]: I need to set it to flex, as I toggle it to display none/flex when the user clicks on the floating button

File: components/expanding-components/section.scss (hunk 3 above)
Comment (mattsskea) [reply components/expanding-components/section.scss:42]: its the same situation, I am toggling display none/flex when users click the button

**Pull request merged**